polaris-intellisense/.yarnrc

```diff
@@ -0,0 +1 @@
+--ignore-engines true
```

Comment:
Unsure what this does... if it's about the engines in package.json this doesn't feel great. Any context?

Reply:
i also have no idea what this does and the only documentation i could find was a [reddit comment](https://www.reddit.com/r/node/comments/o9j87e/what_does_ignoreengines_flag_do/) specifically about npm. Not sure why we would want to use this, it was included in the boilerplate 🤷 

`engines` in package.json is specifically for the version of vscode that this extension can run on, so that doesn't feel related...

Reply:
Lets remove this file.